### DISCUSSION TITLE
Windows runner: Failed to create pod sandbox ... Windows does not support privileged mode

### DISCUSSION THREAD

**Discussion by ghost** [actions/actions-runner-controller#2995] [Questions]
Hey all!

Trying to get a Windows runner working in an EKS kube cluster based off the docs [here](https://github.com/actions/actions-runner-controller/blob/master/docs/configuring-windows-runners.md) - I built the docker image specified on this page and pushed it to my image repo.. the problem is when I create the runnerdeployment as specified on the same page, the pod gets stuck in "ContainerCreating" and I see the following in my events:
`Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create a sandbox for pod "foo-bar-action-runner-deploy-win-zx7q9-gd5n5": Error response from daemon: Windows does not support privileged mode`

.. is the dockerfile on that page not up to date, or has anyone else run into this?

**Comment by kingbj940429** [comment]
I have a same problem..
